LEGACY_SSE_EMULATION.md

```diff
@@ -0,0 +1,512 @@
+                                "bytes": "5a000000909ffe7807b03a5db0c3c183648710db16d408d8425a4e373fc0422a4efed1ab0040bc08786553fcac4521528c9fafca0b0fb86f4c6e9fb9db7a1454dda8ed612c4ea4c9a6378b230ae1e3c236e37d6ebee94339a56cb4be582a",
+                                "parsed": [
+                                    144,
+                                    159,
+                                    254,
+                                    120,
+                                    7
+                                ]
+                            }
+                        ]
+                    ]
+                }
+            },
+            "approvals": [
+                {
+                    "signer": "02022c07e061d6e0b43bbaa25717b021c2ddc0f701a223946a0883b57ae842917438",
+                    "signature": "025d0a7ba37bebe6774681ca5adecb70fa4eef56821eb344bf0f6867e171a899a87edb2b8bf70f2cb47a1670a6baf2cded1fad535ee53a2f65da91c82ebf30945b"
+                }
+            ]
+        }
+    }
+    ```
+    * If the event is a V2 variety - it will be omitted so a 2.x event like:
+    ```
+    {
+        "TransactionAccepted": {
+            "Version1": {
+                ...
+            }
+        }
+    }
+    ```
+    will be omitted from the legacy SSE streams
+
+## Translating `TransactionExpired` event
+  * If it's a Deploy variety it will be unpacked and sent. So a 2.x `TransactionExpired` event:
+  ```json
+    {
+        "TransactionExpired": {
+            "transaction_hash": {
+                "Deploy": "565d7147e28be402c34208a133fd59fde7ac785ae5f0298cb5fb7adfb1b054a8"
+            }
+        }
+    }
+  ```
+  will be sent as a legacy `DeployExpired` event:
+  ```json
+   {
+       "DeployExpired": {
+           "deploy_hash": "565d7147e28be402c34208a133fd59fde7ac785ae5f0298cb5fb7adfb1b054a8"
+       }
+   }
+  ```
+
+  * If it's a Version1 variety it will be omitted from legacy SSE streams. So a 2.x `TransactionExpired` event:
+  ```json
+    {
+        "TransactionExpired": {
+            "Version1": {
+                "hash": "565d7147e28be402c34208a133fd59fde7ac785ae5f0298cb5fb7adfb1b054a8"
+            }
+        }
+    }
+  ```
+    will be omitted
+
+## Translating `TransactionProcessed` event.
+    * If `transaction_hash` field is a `Version1`, the event will be ignored.
+    * If `transaction_hash` field is a `Deploy`, it's value will be used as `DeployProcessed.deploy_hash`
+    * If `initiator_addr` field is not a `PublicKey` type, the event will be omitted.
+    * If `initiator_addr` field is a `PublicKey` type, it's value will be used as `DeployProcessed.account`
+    * `timestamp`, `ttl`, `block_hash` will be filled from analogous fields in the `TransactionProcessed` event
+    * If `execution_result` is a `Version1` type, it's value will be copied as-is do the `DeployProcessed.execution_result` field.
+    * If `execution_result` is a `Version2` type please see (this paragraph)[#translating-executionresultv2]
```

Comment:
```suggestion
    * If `execution_result` is a `Version2` type please see [this paragraph](#translating-executionresultv2)
```

Please also check other links, as this is probably some copy-paste issue.

Also, most likely the spaces before `*` are not needed here, since this breaks the rendering.

Reply:
:+1: